### PR TITLE
Implement exercise list port

### PR DESCRIPTION
**Pull request by jpablo2002**

### CODE REVIEW

File: src/main/java/ca/utoronto/fitbook/adapter/persistence/localmemory/ExerciseLocalMemoryRepository.java [hunk 1]
```diff
@@ -29,4 +32,19 @@ public Exercise getById(String id) throws EntityNotFoundException {
     public void save(Exercise entity) {
         datastore.put(entity.getId(), entity);
     }
+
+    /**
+     * @param exerciseIds The post ids to be fetched
```
Comment (Ludosss) [inline src/main/java/ca/utoronto/fitbook/adapter/persistence/localmemory/ExerciseLocalMemoryRepository.java:37]: rewrite comment to be "exercise ids"

File: src/main/java/ca/utoronto/fitbook/adapter/persistence/firebase/ExerciseFirebaseRepository.java [hunk 2]
```diff
@@ -61,4 +64,18 @@ public Exercise getById(String id) throws EntityNotFoundException {
     public void save(Exercise entity) {
         datastore.getCollection("exercises").document(entity.getId()).set(entity);
     }
+
+    /**
+     * @param exerciseIds The post ids to be fetched
```
Comment (Ludosss) [inline src/main/java/ca/utoronto/fitbook/adapter/persistence/firebase/ExerciseFirebaseRepository.java:69]: rewrite comment to be "exercise ids"

**Review by Ludosss — CHANGES_REQUESTED** [review]
slight comment changes

File: src/main/java/ca/utoronto/fitbook/adapter/persistence/localmemory/ExerciseLocalMemoryRepository.java (hunk 1 above)
Comment (jpablo2002) [reply src/main/java/ca/utoronto/fitbook/adapter/persistence/localmemory/ExerciseLocalMemoryRepository.java]: Good point, resolved!

File: src/main/java/ca/utoronto/fitbook/adapter/persistence/firebase/ExerciseFirebaseRepository.java (hunk 2 above)
Comment (jpablo2002) [reply src/main/java/ca/utoronto/fitbook/adapter/persistence/firebase/ExerciseFirebaseRepository.java]: Good point, resolved!

**Review by Ludosss — APPROVED** [review]
very nice

**Review by youssefsoli — APPROVED** [review]
Simple and short, looks good r+

**Review by modaserMoj — APPROVED** [review]
Implemented some changes to the exercise repo that helps members

**Pull request merged**